launchdarkly/ios-client
# updates to version 2.9.1

Fixes a Carthage build issue that could cause the sdk to crash at runtime. A corresponding CocoaPods change keeps the version numbers up-to-date.

## Testing steps
In the hello-tvOS project directory
1. Delete `hello-tvOS/Carthage`
2. edit `Cartfile` replacing the existing git reference with `git "file:///Path/to/ios-client" "version_2.9.1"
` (Note the 3 `/` in the path)
3. from the command line in the hello-tvOS project directory, run `carthage update`. Verify the build succeeds. 
4. Open `hello-tvOS/Carthage/Build`. Verify each OS has a newly built `Darkly.framework` and `DarklyEventSource.framework`
5. Open hello-tvOS.xcodeproj.
6. Run on any of the tv simulators. Verify the app does not crash.
